### ISSUE TITLE
Inserts on tables with computed columns

### ISSUE THREAD

**Issue by mkp-incom** [ultorg/public_issues#35]
Doing inserts on tables that contain computed columns fails:

1. If the column is not hidden and left blank you get:
![image](https://user-images.githubusercontent.com/79151501/188856121-95e742aa-6887-40d7-abf6-758ee28ba2a0.png)
2. If the column is hidden you get:
![image](https://user-images.githubusercontent.com/79151501/188856602-6fbad879-4d8e-4384-9236-eff8ca9471c1.png)

Attaching code for the table-schema used in this context:
`CREATE TABLE ultorgtest.testtable(
	[Id] [int] IDENTITY(1,1) NOT NULL,
	[Code]  AS (CONVERT([nvarchar](128),([SystemName]+'-')+CONVERT([nvarchar](7),[Id]))) PERSISTED NOT NULL,
	[SystemName] [nvarchar](96) NOT NULL,
	[Name] [nvarchar](128) NOT NULL,
	[CreatedBy] [nvarchar](100) NULL,
	[ModifiedBy] [nvarchar](100) NULL,
	[CreatedOn] [datetime] NOT NULL,
	[ModifiedOn] [datetime] NOT NULL
	 CONSTRAINT [PK_testcode] PRIMARY KEY CLUSTERED 
(
	[Code] ASC
)
)`

**Comment by mkp-incom** [comment]
Forgot to mention this is on a Azure SQL Database

**Comment by eirikbakke** [comment]
Hi, Martin. Thanks for the example CREATE TABLE statement!

This is a missing feature that I hadn't gotten around to implementing yet: The ability to accept auto-incremented keys (replacing the "+1", "+2" etc. indications) during INSERT statements. I can add that.

(Another subtlety here is the difference between setting a column explicitly to NULL vs. omitting it from the INSERT clause. The difference wasn't shown in the user interface anywhere, but became evident when you got slightly different error messages in the two cases (probably in the first case you had pressed DELETE on the "+1" value). I might just always omit columns with a null value from the INSERT clause, to let them populate with any auto-incremented or calculated values.)

**Comment by mkp-incom** [comment]
Hi Eirik, perhaps the explicit option to define a column as "Read-Only" would be an option for users to define "omit insert/updates" functionality, preferably visible in the UI (like toned / greyed out columns). Looking forward to this feature :-)

**Comment by eirikbakke** [comment]
Hi, Martin, thanks for the chat today!

The issue with INSERTs on tables with auto-generated keys (and computed columns) has been fixed in Ultorg 1.2.1. Existing users can find the latest release at their old download links.

I created a [separate issue](https://github.com/ultorg/public_issues/issues/40) for the proposed feature of being able to define certain columns as read-only on the client side.
